mask/eolearn/tests/test_cloud_mask.py

```diff
@@ -10,11 +10,34 @@
 """
 
 import numpy as np
+import pytest
 from numpy.testing import assert_array_equal
 from pytest import approx
 
 from eolearn.core import FeatureType
 from eolearn.mask import CloudMaskTask
+from eolearn.mask.cloud_mask import _get_window_indices
+
+
+@pytest.mark.parametrize(
+    "num_of_elements, middle_idx, window_size, expected_indices",
+    (
+        [100, 0, 10, (0, 10)],
+        [100, 1, 10, (0, 10)],
+        [100, 50, 10, (45, 55)],
+        [100, 99, 10, (90, 100)],
+        [100, 100, 10, (90, 100)],
+        [100, 0, 11, (0, 11)],
+        [100, 1, 11, (0, 11)],
+        [100, 50, 11, (45, 56)],
+        [100, 99, 11, (89, 100)],
+        [100, 100, 11, (89, 100)],
```

Comment:
does this half just repeat with a different window?
I also don't see a case where `window_size` is larger than the number of elements.
Lastly, try adjusting `number_of_elements` to something less 'pretty' in some cases

Reply:
> does this half just repeat with a different window?

I just wanted to add tests for both odd and even window sizes. Would you prefer I implement it differently? Perhaps there is no need to do it at such extent?

> I also don't see a case where window_size is larger than the number of elements.

Yeah, forgot to re-add it after updating. Updated now along with the last remark.

Reply:
i usually mix the cases up quite a bit.
In cases where we want to test a lot of options we could use [hypothesis](https://hypothesis.readthedocs.io/en/latest/) and automate that a bit :thinking: certainly something to look.

Resolving this one, if i find an issue in the test cases i'll open a new thread